agent.py

```diff
@@ -994,7 +994,7 @@ def findTribe(self):
         sugarscape = self.cell.environment.sugarscape
         numTribes = sugarscape.configuration["environmentMaxTribes"]
         zeroes = 0
-        tribeCutoff = math.floor(len(self.tags) / numTribes)
+        tribeCutoff = round(len(self.tags) / numTribes)
```

Comment:
Double check this math (and related changes). With 10 tribes and 250 agents, I am getting one tribe with a single member, one tribe with no members, and other tribes eat up the rest of the agents.

Reply:
Should be fixed.